### ISSUE TITLE
Drop-down menu won't change themes?

### ISSUE THREAD

**Issue by kyoogoo** [company-mode/company-mode#697]
Per the title of this inquiry, the drop-down menu has an unpleasant aesthetic. Previously in my myinit.org, I incorporated this string of code to change the color scheme for company, which according to the EmacsWiki: Company-Mode adjusts the color scheme for dark backgrounds (I have removed this portion of code from myinit.org):

`(require 'color)
(let ((bg (face-attribute 'default :background)))
(custom-set-faces
 `(company-tooltip ((t (:inherit default :background ,(color-lighten-name bg 2)))))
 `(company-scrollbar-bg ((t (:background ,(color-lighten-name bg 10)))))
 `(company-scrollbar-fg ((t (:background ,(color-lighten-name bg 5)))))
 `(company-tooltip-selection ((t (:inherit font-lock-function-name-face))))
 `(company-tooltip-common ((t (:inherit font-lock-constant-face))))))`

However, since following this block of code, my emacs has a strange appearance now:

<img width="327" alt="screen shot 2017-08-18 at 12 43 29 am" src="https://user-images.githubusercontent.com/18727118/29479672-a7409786-8441-11e7-9558-bddcbc781e3a.png">

I'm currently using the Atom-One-Dark-Theme, and it incorporates its own tooltip theme. However, it will not change the tooltip/drop-down menu to the proper colors. Here is the following code in the atom-one-dark-theme.el:

`;; company-mode
   `(company-tooltip ((t (:foreground ,atom-one-dark-fg :background ,atom-one-dark-bg-1))))
   `(company-tooltip-annotation ((t (:foreground ,atom-one-dark-mono-2 :background ,atom-one-dark-bg-1))))
   `(company-tooltip-selection ((t (:foreground ,atom-one-dark-fg :background ,atom-one-dark-gray))))
   `(company-tooltip-mouse ((t (:background ,atom-one-dark-gray))))
   `(company-tooltip-common ((t (:foreground ,atom-one-dark-orange-2 :background ,atom-one-dark-bg-1))))
   `(company-tooltip-common-selection ((t (:foreground ,atom-one-dark-orange-2 :background ,atom-one-dark-gray))))
   `(company-preview ((t (:background ,atom-one-dark-bg))))
   `(company-preview-common ((t (:foreground ,atom-one-dark-orange-2 :background ,atom-one-dark-bg))))
   `(company-scrollbar-fg ((t (:background ,atom-one-dark-mono-1))))
   `(company-scrollbar-bg ((t (:background ,atom-one-dark-bg-1))))`

Sorry if there's some ambiguity in this post; I'll try to clear out any questions that you might have.

**Comment by dgutov** [comment]
> Atom-One-Dark-Theme, and it incorporates its own tooltip theme. However, it will not change the tooltip/drop-down menu to the proper colors

Have you tried filing a bug?

**Comment by kyoogoo** [comment]
Yes, I have filed it as a bug, but this has been occurring with other themes as well. Perhaps it's an issue with my init file?

**Comment by dgutov** [comment]
> Perhaps it's an issue with my init file?

Might be. You can run `emacs -Q` (to skip the init file), to figure that out.
